Lib/http/server.py

```diff
@@ -1040,6 +1056,10 @@ def _main(args=None):
         except OSError as e:
             parser.error(f"Failed to read TLS password file: {e}")
 
+    response_headers = []
+    for header, value in args.header or []:
+        response_headers.append((header, value))
+
```

Comment:
Does this need to be a list of tuples? 

Or can it be a list of lists? If so, we can use `args.header` directly:
```suggestion
```